.gitignore

```diff
@@ -158,3 +158,4 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+/.idea/
```

Comment:
pick one

Reply:
Addressed in commit - bcb84a552552cb710f87486bbee433af62b38f6d.